city_scrapers/spiders/tulok_boed.py

```diff
@@ -0,0 +1,80 @@
+import json
+from datetime import datetime
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class TulokBoedSpider(CityScrapersSpider):
+    name = "tulok_boed"
+    agency = "Tulsa Public Schools Board of Education"
+    timezone = "America/Chicago"
+    start_urls = [
+        "https://tulsaschools.diligent.community/Services/MeetingsService.svc/meetings?from=2024-12-01&to=9999-12-31"  # noqa
+    ]
+    agenda_url = "https://tulsaschools.diligent.community/Portal/MeetingInformation.aspx?Org=Cal&Id={}"  # noqa
+    custom_settings = {"ROBOTSTXT_OBEY": False}
+
+    def parse(self, response):
+        """
+        Parse meeting items from the Diligent Community API.
+        Returns JSON array of meeting objects with full details.
+        """
+        data = response.json()
+        for item in data:
+            meeting = Meeting(
+                title=self._parse_title(item),
```

Comment:
> {"title": "Regular Meeting - Dec 14 2026 (DRAFT AGENDA)",

The meeting title doesn't need to show the date or the file type. There is a field in the API response called `MeetingTypeName`, you can use that in here.

```suggestion
                title=item.get("MeetingTypeName"),
```
And remove the `_get_parse` function.